postcss.config.js

```diff
@@ -0,0 +1,12 @@
+const DEBUG = process.env.NODE_ENV !== 'production'
+
+module.exports = {
+  plugins: [
+    require('stylelint')(),
+    require('postcss-import')(),
+    require('postcss-cssnext')(),
+  ].concat(DEBUG ? [
+    require('postcss-reporter')(),
+    require('postcss-browser-reporter')(),
+  ] : []),
+}
```

Comment:
PostCSS config is now externally loaded from `postcss.config.js`. It doesn't live inside webpack.config.js anymore.